handlers/restart_service.yml

```diff
@@ -6,6 +6,9 @@
     state: restarted
   become: true
 
+- name: Wait for Jenkins to bind to TCP port before checking HTTP
```

Comment:
Just to keep things succinct, could you please rename this to "Wait for Jenkins to Open Port"? Thanks!